modules/galaxy/tasks/milkywayconversiontask.cpp

```diff
@@ -22,30 +22,53 @@
  * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
  ****************************************************************************************/
 
-#include <apps/DataConverter/milkywayconversiontask.h>
+#include <modules/galaxy/tasks/milkywayconversiontask.h>
 #include <modules/volume/textureslicevolumereader.h>
 #include <modules/volume/rawvolumewriter.h>
 #include <modules/volume/volumesampler.h>
 
 namespace openspace {
-namespace dataconverter {
     
-MilkyWayConversionTask::MilkyWayConversionTask(
-    const std::string& inFilenamePrefix,
-    const std::string& inFilenameSuffix,
-    size_t inFirstIndex,
-    size_t inNSlices, 
-    const std::string& outFilename,
-    const glm::ivec3& outDimensions)
-    : _inFilenamePrefix(inFilenamePrefix)
-    , _inFilenameSuffix(inFilenameSuffix)
-    , _inFirstIndex(inFirstIndex)
-    , _inNSlices(inNSlices)
-    , _outFilename(outFilename)
-    , _outDimensions(outDimensions) {}
+MilkywayConversionTask::MilkywayConversionTask(const ghoul::Dictionary& dictionary) {
+    std::string inFilenamePrefix;
+    if (dictionary.getValue("InFilenamePrefix", inFilenamePrefix)) {
```

Comment:
I'd prefer these constants to be defined at the top of the file in an anonymous namespace